sdks/python/apache_beam/examples/wordcount_xlang_sql.py

```diff
@@ -15,12 +15,16 @@
 # limitations under the License.
 #
 
-"""A cross-language word-counting workflow."""
+"""A cross-language word-counting workflow.
+
```

Comment:
Can we also briefly describe what this pipeline does and what output it produces ?

Reply:
Wordcount is the quintessential example of all Beam pipelines, and this does exactly the same thing that all the other wordcount variants in this folder do. I think it makes more sense to highlight the differences.